mmv1/templates/terraform/custom_delete/firestore_field_delete.go.erb

```diff
@@ -53,4 +53,7 @@ if err != nil {
 }
 
 log.Printf("[DEBUG] Finished deleting Field %q", d.Id())
+
+d.SetId("") // Remove the resource from state
```

Comment:
I'll remove this as, while it might be a legitimate thing to add, this PR isn't the correct place